CHANGELOG.md

```diff
@@ -16,6 +16,20 @@ Sections
 ### Developers
 -->
 
+## [2.2.3] - 2018-10-05
+
+### Added
+- Added `NeoPixelsLightStrip` accessory. [#144](https://github.com/ikalchev/HAP-python/pull/144)
+- Added new Accessory categories. [Commit](https://github.com/ikalchev/HAP-python/commit/dbf1d5d8fea814af52098f3a2f3cd5a47cd889c1)
```

Comment:
If you first push the new accessory categories, we could update this PR and then merge it.